blog_post.md

```diff
@@ -153,7 +153,19 @@ We thus compare 4 different setups for each environment: without tricks (vanilla
 We run each experiment with 25 different random seeds to achieve more statistically sound results, while taking into account our computational budget. If the maximal absolute Q-value predicted in any of the last 20 episodes is above the threshold $$\frac{1}{1-\gamma}$$, we say soft divergence occurs. 
 <!--- At the end, we compare the configurations by counting how many times each of them has diverged. -->
 
-Even though the original paper uses a convolutional neural network to play Atari games, we focus on the simpler and less computationally expensive [Cart Pole](https://gym.openai.com/envs/CartPole-v1/), [Mountain Car](https://gym.openai.com/envs/MountainCar-v0/) and [Acrobot](https://gym.openai.com/envs/Acrobot-v1/) (from [OpenAI gym](https://gym.openai.com/)). Our model is a single-layer fully-connected network with a hidden layer of size 128. 
+Even though the original paper uses a convolutional neural network to play Atari games, we limit ourselves to simpler environments so that we can use a fully-connected neural network with a single layer of size 128. We pick the classical control theory environments made available by [OpenAI gym](https://gym.openai.com/envs/#classic_control).
+
+### [Cart Pole](https://gym.openai.com/envs/CartPole-v1/)
+![cartpole](img/cartpole.gif)
+The agent tries to balance a pole on a cart by applying a rightward or a leftward force. For every time step the pole remains upright (less than 15 degrees from vertical), the agent recieves a reward of +1. This problem is very easy to solve. Therefore, we chose this as a representative of problems with low-level difficulty.
```

Comment:
```suggestion
In the Cart Pole environment, the agent tries to balance a pole on a cart by applying a rightward or a leftward force to the cart. For every time step the pole remains upright (less than 15 degrees from vertical), the agent receives a reward of +1. Since his problem is considered relatively easy to solve. we chose it as a representative of problems with low difficulty.
```

Reply:
remove "to the cart"